dts: bindings: pwm: nxp,imx-pwm: add PWM period cell

gmarull opened the pull request:
The PWM period cell will soon be required by the pwm_dt_spec facilities,
this patch adds support for it.

Prep work for https://github.com/zephyrproject-rtos/zephyr/pull/44523

gmarull committed:
dts: bindings: pwm: nxp,imx-pwm: add PWM period cell

The PWM period cell will soon be required by the pwm_dt_spec facilities,
this patch adds support for it.

Signed-off-by: Gerard Marull-Paretas <gerard.marull@nordicsemi.no>

gmarull committed:
boards: arm: mimxrt1064/1160/1170_evk: add PWM period cell

Add the PWM period cell to PWM driven LEDs. A value of 20 msec has been
chosen as it is the most common value used in other boards.

Signed-off-by: Gerard Marull-Paretas <gerard.marull@nordicsemi.no>

anangl commented on boards/arm/mimxrt1064_evk/mimxrt1064_evk.dts:
```diff
@@ -52,7 +52,7 @@
 
 		green_pwm_led: green_pwm_led {
-			pwms = <&flexpwm2_pwm3 0>;
+			pwms = <&flexpwm2_pwm3 0 PWM_MSEC(20)>;
```
I think you need to do the same in
https://github.com/zephyrproject-rtos/zephyr/blob/8aece303cac08ff6d6c82cd25e2b14b7f827e9ea/boards/arm/mimxrt1160_evk/mimxrt1160_evk.dtsi#L35
and
https://github.com/zephyrproject-rtos/zephyr/blob/8aece303cac08ff6d6c82cd25e2b14b7f827e9ea/boards/arm/mimxrt1170_evk/mimxrt1170_evk.dtsi#L35

gmarull replied:
done

gmarull committed:
dts: arm: nxp: nxp_rt/rt11xx: include PWM dt-bindings by default

In order to be consistent with other platforms, include the dt-bindings
for PWM.

Signed-off-by: Gerard Marull-Paretas <gerard.marull@nordicsemi.no>
